packages/renderer/src/lib/chat/components/suggested-actions.svelte

```diff
@@ -104,23 +104,49 @@ async function onclick(suggestedAction: SuggestedAction): Promise<void> {
         throw Error(`Suggested action ${suggestedAction.action} requires MCP with id ${mcpId} but it was not found.`);
       }
 
-      acc.push([mcpInfo.name, tools]);
+      acc.push([mcpInfo.name, tools, server?.id]);
       return acc;
     },
-    [] as Array<[string, string[]]>,
+    [] as Array<[string, string[], string | undefined]>,
```

Comment:
suggestion(non-blocking): use an object instead of a tuple to name properties

see https://github.com/gastoner/kortex/pull/1/changes for the implemented suggestion